project1.html

```diff
@@ -0,0 +1,155 @@
+
+<button type="button">html</button>
+<button type="button">css</button>
+<button type="button">javaScript</button>
+
+
+<button type="button">See Project</button>
+    
+</section>
+-->
+
+<section>
+
+<div class="flex-container1">
+
+<div class="snapshoot">
+  <a><img src="Snapshoot Portfolio.png" alt=" "></a>
+</div>
+
+<div class="flex-container12">
+<div class="one1"><h1>Multi-Post Stories</h1>
+</div>
+
+<div class="two2">
+
+<div> CANOPY </div>
+<div> </div>
+<div> Back End Dev </div>
+<div> </div>
+<div> 2015 </div>
+
+</div>
+
+<div class="three3"><p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p></div>
+
+<div class="four4">
+<div><button>html</button></div>
+<div></div>
+<div></div>
+<div><button>css</button></div>
+<div></div>
+<div></div>
+<div><button>JavaScript</button></div>
+</div>
+
+<div class="five5"><button>See Project</button></div>
+
+</div>
+</div>
+
+<div class="flex-container">
+<div><h1>About Myself</h1></div>
+
+  <div><p>I'm a software developer! I can help you build a product, feature or website
+    Look through some of my work and experience! If you like what you see and have
+    a project you need coded, don't hesitate to contact me.</p></div>
+
+   <div class="one">LET'S CONNECT </div>
+
+      <a class="github"><img src="github.jpg" alt=" "></a>
+      <a class="linkedin"><img src="Linkedinicon.jpg" alt=" "></a>
+      <a class="angellist"><img src="angellist.jpg" alt=" "></a>
+      <a class="twitter"><img src="twitter.jpg" alt=" "></a>
+      <a class="medium"><img src="medium.jpg" alt=" "></a>
+
+      <div><button class="two">Get my resume</button></div>
+</div>
+
+<div class="flex-container2">
+
+  <div><h4 class="languages">Languages</h4></div>
+
+<div class="flex-container">
+
+  <div class="flex-container3">
+  <div><a><img src="Ellipse 1.jpg" alt="Javascript"></a></div>
+  <div><h5> Javascript</h5></div>
+  <div> </div>
+  <div> </div>
+  <div> </div>
+  <div> </div>
+  <div> </div>
+  </div>
+
+  <div class="flex-container4">
+  <div><a><img src="Ellipse 1 (1).jpg" alt="HTML"></a></div>
+  <div><h5>HTML</h5></div>
+  <div> </div>
+  <div> </div>
+  <div> </div>
+  <div> </div>
+  <div> </div>
+  </div>
+
+  <div class="flex-container5">
+    <div><a><img src="Ellipse 1 (2).jpg" alt="CSS"></a></div>
+    <div><h5>CSS</h5></div>
+    <div> </div>
+  <div> </div>
+  <div> </div>
+  <div> </div>
+  <div> </div>
+  </div>
+ 
+</div>
+
+ <div> 
+
+    <a class="Vector1"><img src="Vector1.png" alt=""></a>
+    <a class="Vector2"><img src="Vector2.png" alt=""></a>
+    <a class="Vector3"><img src="Vector2.png" alt=""></a>
+
+    <div><h4 class="Frameworks">Frameworks</h4></div>
+
+  <a class="Separator1"><img src="Separator Bottom.jpg" alt=""></a></div>
+
+  <div><h4 class="Skills">Skills</h4></div>
+
+    <a class="Separator2"><img src="Separator Bottom.jpg" alt=""></a>
+  
+    <a class="Separator1"><img src="Separator Bottom.jpg" alt=""></a>
+
+</div>    
```

Comment:
- [ ] [REQUIRED] Please make changes! The technology categories ("Languages", "Frameworks" and "Skills") should be a list `ul` and each of them a `li`.